components/stats/Chart.js

```diff
@@ -204,26 +267,124 @@ export default class Chart extends React.Component {
+    //let bufferY = (600 - yCeiling) / 2;
+    let bufferX = (600 - xWall) / 2;
+    const beginX = (Math.floor(minX) + bufferX) * dX
+    const endX = (Math.floor(maxX) - bufferX) * dX
+    let spacing = (beginX - endX)/gridLines;
+    const gLines = [];
+
+    for(let gridX = beginX; gridX > endX; gridX+= spacing){
+      gLines.push(gridX);
+      console.log(gridX);
+    }
+
+    return null;
+  } */
+
+  createCircles = () => {
+    const organizedData = this.state.organizedData;
+    let oData = organizedData.flat(2);
+    let allCircles = oData.map((g, index) => {
       return (
-        <React.Fragment>
-          <CreateChart
-            minX={this.state.minX}
-            maxX={this.state.maxX}
-            minY={this.state.minY}
-            maxY={this.state.maxY}
-            deltaX={this.state.deltaX}
-            deltaY={this.state.deltaY}
-            ticks={this.state.ticks}
-            xLabel={this.state.xLabel}
-            yLabel={this.state.yLabel}
-            organizedData={this.state.organizedData}
-          />
-        </React.Fragment>
+        <circle key={index} cx={g.x} cy={g.y} r="2" css={STYLES_CHART_CIRCLE} />
+      );
+    });
+    ReactDOM.render(allCircles, document.getElementById("circles"));
+  };
+
+  createLines = () => {
+    const organizedData = this.state.organizedData;
+
+    if (organizedData.length) {
+      let allLines = [];
+      for (let groups of organizedData) {
+        for (let group of groups) {
+          let polyLines = this.loopData(group);
+          allLines.push(polyLines);
+        }
+
+        ReactDOM.render(allLines, document.getElementById("lines"));
+      }
+    }
+  };
+
+  loopData = (g) => {
+    let coordinates = [];
+    let i = {};
+    g.map((o, index) => {
+      coordinates.push(o.x);
+      coordinates.push(o.y);
+      i[`id`] = o.id;
+    });
+    let polyLine = (
+      <polyline
+        css={STYLES_CHART_LINE}
+        key={i.id}
+        points={this.drawPoints(coordinates)}
+      />
+    );
+    return polyLine;
+  };
+
+  drawPoints = (a) => {
+    const c = a.toString();
+    const regex = /([0-9]+),\s([0-9]+),\s/g;
+    const cOrganized = c.replace(regex, "$1,$2 ");
+    return cOrganized;
+  };
+
+  createTicks = () => {
+    const ticks = this.state.ticks;
+    const fTicks = ticks.flat(1);
+    const tickLines = [];
+
+    for (let tick of fTicks) {
+      const tDate = new Date(tick.date);
+      const month = this.monthNames[tDate.getMonth()];
+      const year = tDate.getUTCFullYear();
+
+      let tickLine = (
+        <g>
+          <line css={STYLES_X_LINE} x1={tick.x} y1="550" x2={tick.x} y2="560" />
+          <text css={STYLES_CHART_TEXT} textAnchor="middle" x={tick.x} y="575">
+            {`${month} ${year}`}
+          </text>
+        </g>
       );
-    } else {
-      return <React.Fragment></React.Fragment>;
+      tickLines.push(tickLine);
     }
+    ReactDOM.render(tickLines, document.getElementById("tickContainer"));
```

Comment:
You don't need `ReactDOM`, You can just use `this.state.ticks.map(each => {})` to achieve your goals and declare that inside of `render` I believe.